plugins/XmlMaterialProfile/XmlMaterialProfile.py

```diff
@@ -439,6 +439,8 @@ def deserialize(self, serialized):
         meta_data["base_file"] = self.id
         meta_data["status"] = "unknown"  # TODO: Add material verfication
 
+        global_setting_values = {}
```

Comment:
Global?! You should be changing only the settings of the extruder in which this material is loaded!

This is either a bug or a rather unfortunate name for a variable.... Or is it?